### PR TITLE
Added disclaimer for extensions

### PR DESCRIPTION
**Pull request by jeluard**
### Summary:

Inform users of current risks with extensions usage.
Move to release version of pluto.
Disable `list` component

status: ready

### CODE REVIEW

**Review by janherich — APPROVED** [review]

File: src/status_im/extensions/core.cljs [hunk 1]
```diff
@@ -142,7 +142,7 @@
                 'input              {:value input :properties {:on-change :event :placeholder :string}}
                 'button             {:value button :properties {:on-click :event}}
                 'link               {:value link :properties {:uri :string}}
-                'list               {:value list :properties {:data :vector :item-view :view}}
+                ;'list               {:value list :properties {:data :vector :item-view :view}}
```
Comment (flexsurfer) [inline src/status_im/extensions/core.cljs:146]: no list ?

**Review by flexsurfer — APPROVED** [review]

File: src/status_im/extensions/core.cljs (hunk 1 above)
Comment (jeluard) [reply src/status_im/extensions/core.cljs:146]: It's broken somehow, no time to fix

**Review by alwx — APPROVED** [review]

File: src/status_im/extensions/core.cljs (hunk 1 above)
Comment (flexsurfer) [reply src/status_im/extensions/core.cljs:146]: ok

**Comment by asemiankevich** [comment]
@jeluard it would be great if you can add a detailed description of what is added and where. A screenshot in this particular case and when it should appear.

**Comment by jeluard** [comment]
@asemiankevich Sorry! It's just a simple disclaimer in the screen that pops up when you install an extension (with the `INSTALL` button)

**Comment by asemiankevich** [comment]
@jeluard please follow our guidelines and adjust this error message: `white text on the red popup`. You can take some inspiration from wallet or pass phrase validation pop ups 

image

**Comment by asemiankevich** [comment]
what is the difference? 

image

**Comment by asemiankevich** [comment]
image

**Comment by asemiankevich** [comment]
lets merge this and have it fixed later

**Comment by chadyj** [comment]
Did you cherry pick this for the release branch?

**Comment by jeluard** [comment]
@chadyj yes!

**Pull request merged**